### PR TITLE
Update sensor.spotcrime.markdown

### PR DESCRIPTION
**Pull request by jcconnell**
Changes the documentation to reflect the user supplied API key requirement.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12926

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.spotcrime.markdown [hunk 1]
```diff
@@ -18,23 +18,35 @@ The `spotcrime` sensor allows one to track reported incidents occurring in a giv
...
+  type: string
+radius:
+  description: Radius you'd like to search within. 0.01 = 1 mile.
+  required: true
+  type: float
+api_key:
+  description: The API key to access the service.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (fabaff) [inline source/_components/sensor.spotcrime.markdown]: `include` and `exclude` are missing in the description.

File: source/_components/sensor.spotcrime.markdown [hunk 2]
```diff
@@ -18,23 +18,35 @@ The `spotcrime` sensor allows one to track reported incidents occurring in a giv
...
+
 ```yaml
 sensor:
   - platform: spotcrime
     name: <any name>
-    radius: <your radius>
     days: <your days>
+    radius: <your radius>
+    api_key: <"your_api_key_here">
+    include:
```
Comment (fabaff) [inline source/_components/sensor.spotcrime.markdown]: Don't add optional keys to the sample. We try to keep them minimal.

**Comment by jcconnell** [comment]
Sorry, looks like I got the two sections reversed in regards to optional vs. required keys.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**